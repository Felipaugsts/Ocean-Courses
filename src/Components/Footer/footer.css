.footer { 
    background-color: #ECECEC;
    width: 100%;
    height: 275px;
    margin-top: 30px !important;
    display: flex;
    justify-content: space-evenly;
    align-items: center;
    flex-direction: column;

}
.logo { 
    height: 30px;
    
}