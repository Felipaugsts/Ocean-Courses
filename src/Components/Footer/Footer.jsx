import './footer.css'
import logo from '../../Assets/Logo/logo.png'

const Footer = () => { 
    return ( 
        <div className='footer'>
            <img className='logo' src={logo} />
            <p className='medium dark light'>
                Lorem ipsum dolor, sit amet consectetur adipisicing elit. <br/>
                 Impedit dolorum quo repellendus magni, officia ea mollitia asperiores! <br/>
                  Aliquam explicabo perferendis quis nisi quaerat unde accusamus laborum <br/>
                  cupiditate quae sapiente? Deserunt?
            </p>

            <p className='small gray'> copyright samsung electronics 2022. all rights reserved </p>
        </div>
    )
}
export default Footer;