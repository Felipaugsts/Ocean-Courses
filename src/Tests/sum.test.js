import renderer from 'react-test-renderer';
import Navbar from '../Components/Navigation/Navigation'
import Footer from '../Components/Footer/Footer'


test('testing jest', () => {
  expect(true).toBe(true);
});

// SECTION:  Testing Components 

it(' should render correctly - Navigation bar', () => {
  const tree = renderer
    .create(<Navbar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it(' should render correctly - Footer', () => {
  const tree = renderer
    .create(<Footer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
