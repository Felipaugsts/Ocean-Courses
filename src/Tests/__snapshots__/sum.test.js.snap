// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render correctly - Footer 1`] = `
<div
  className="footer"
>
  <img
    className="logo"
    src="test-file-stub"
  />
  <p
    className="medium dark light"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
    <br />
    Impedit dolorum quo repellendus magni, officia ea mollitia asperiores! 
    <br />
    Aliquam explicabo perferendis quis nisi quaerat unde accusamus laborum 
    <br />
    cupiditate quae sapiente? Deserunt?
  </p>
  <p
    className="small gray"
  >
     copyright samsung electronics 2022. all rights reserved 
  </p>
</div>
`;

exports[` should render correctly - Navigation bar 1`] = `
<div
  className="navigation"
>
  <div
    className="navigation-bar"
  >
    <img
      className="navigation-logo"
      src="test-file-stub"
    />
    <div
      className="navigation-option dark-blue w-normal"
    >
      <p>
        Trilhas
      </p>
      <p>
        Cursos
      </p>
      <p>
        Agenda
      </p>
    </div>
    <p
      className="dark-blue w-bold"
    >
      Entrar/Cadastrar
    </p>
  </div>
  <div
    className="icon-wrapper"
  >
    <i
      className="icon bi bi-list"
    />
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="navigation"
>
  <div
    className="navigation-bar"
  >
    <img
      className="navigation-logo"
      src="test-file-stub"
    />
    <div
      className="navigation-option dark-blue w-normal"
    >
      <p>
        Trilhas
      </p>
      <p>
        Cursos
      </p>
      <p>
        Agenda
      </p>
    </div>
    <p
      className="dark-blue w-bold"
    >
      Entrar/Cadastrar
    </p>
  </div>
  <div
    className="icon-wrapper"
  >
    <i
      className="icon bi bi-list"
    />
  </div>
</div>
`;
